technical-reference/reference-technical-articles/technical-doc-xvs-bridge.md

```diff
@@ -0,0 +1,221 @@
+
+- The XVS token contract is deployed on destination/Destination Chains, and it is used within the XVSProxyDest contract.
+- The XVS Token follows the ERC20 standard and extends the `TokenController` ownable contract, which contains all controlling mechanisms of the XVS.
+- It is responsible for setting minting limits for the minter (in this case, the remote bridge contract).
+- When receiving transactions and tokens from the source chain's Bridge contract, an external call is made to mint tokens for the receiver.
+- When sending tokens to the source chain's Bridge contract, an external call is made from the bridge contract to burn tokens from the sender.
+- Offers a blacklisting feature to prevent certain users from receiving XVS tokens.
+- ACM integration is used for setting minting caps and blacklisting, and these settings can be configured via multisig now and VIPs in the near future.
+
+## 8. Variables and Configuration <a name="variables-and-configuration"></a>
+
+Here, we provide details about the key variables and configuration settings used in the Bridge:
+
+- `address public accessControlManager`: Stores the address of the access control manager contract. This contract manages permissions for who can call certain functions.
+- `ResilientOracleInterface public oracle`: Represents the interface to interact with the ResilientOracle contract. This contract is responsible for providing price information, especially related to token values.
+- `mapping(uint16 => uint256) public chainIdToMaxSingleTransactionLimit`: Maps a destination chain ID to the maximum limit (in USD) for a single transaction to that chain.
+- `mapping(uint16 => uint256) public chainIdToMaxSingleTransactionReceiveLimit`: Maps a destination chain ID to the maximum limit (in USD) for a single transaction to receive from that chain.
+- `mapping(uint16 => uint256) public chainIdToMaxDailyLimit`: Maps a destination chain ID to the maximum limit (in USD) for the total value of transactions in a 24-hour window to that chain.
+- `mapping(uint16 => uint256) public chainIdToMaxDailyReceiveLimit`: Maps a destination chain ID to the maximum limit (in USD) for the total value of transactions in a 24-hour window that can be received from that chain.
+- `mapping(uint16 => uint256) public chainIdToLast24HourTransferred`: Maps a destination chain ID to the total value of transactions transferred to that chain in the current 24-hour window.
+- `mapping(uint16 => uint256) public chainIdToLast24HourReceived`: Maps a destination chain ID to the total value of transactions received from that chain in the current 24-hour window.
+- `mapping(uint16 => uint256) public chainIdToLast24HourWindowStart`: Maps a destination chain ID to the timestamp when the current 24-hour window started for transactions to that chain.
+- `mapping(address => bool) public whitelist`: A mapping of addresses to a boolean indicating whether the address is whitelisted, exempting it from certain limits.
```

Comment:
I don't think we need this section. We'll should include the natspec autogenerated pages